app/Makefile

```diff
@@ -1,15 +1,21 @@
+all:
+	@echo "run:			meteor npm start"
+	@echo "parse-server:		meteor npm run server"
+	@echo "run-test:		meteor npm run test"
+	@echo "test-watch:		meteor npm run test-watch"
+
 # Dev server
 run:
-	npm run start
+	meteor npm start
```

Comment:
I noticed you dropped the `run` from this line while you kept it in the other lines.  Was this intentional?